apps/obsidian/src/components/NodeTypeSettings.tsx

```diff
@@ -75,6 +93,23 @@ const FIELD_CONFIGS: Record<EditableFieldKey, BaseFieldConfig> = {
     type: "color",
     required: false,
   },
+  tag: {
+    key: "tag",
+    label: "Node tag",
+    description: "Tags that signal a line is a node candidate",
+    type: "text",
+    required: false,
+    validate: (value: string) => {
+      if (!value.trim()) return { isValid: true };
+      if (!value.startsWith("#")) {
```

Comment:
nit: tag shouldn't have to start with `#`, we should just take care of that for them.  But as long as there is the error message, this is works.